Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -284,6 +292,75 @@ public final class ViewAnnotationManager {
         observers.removeValue(forKey: ObjectIdentifier(observer))
     }
 
+    // MARK: Framing
+
+    /// Sets the visible region so that the map displays the specified annotations.
+    ///
+    /// - Parameter ids: The list of annotations ids to be framed.
+    /// - Parameter padding: The padding to be set around the edges of the map view, default is `UIEdgeInsets.zero`.
+    /// - Parameter pitch: Default is 0.
+    /// - Parameter animationDuration: Default is 1.
+    public func showAnnotations(
+        ids: [String],
+        padding: UIEdgeInsets = .zero,
+        pitch: CGFloat? = nil,
+        animationDuration: TimeInterval = 1
+    ) {
+        let options = ids.compactMap { try? mapboxMap.options(forViewAnnotationWithId: $0) }
+        guard !options.isEmpty else { return }
+
+        var top, left, bottom, right: ViewAnnotationOptions!
+        for annotationOption in options where annotationOption.point != nil {
+            if top == nil || top.point.coordinates.latitude < annotationOption.point.coordinates.latitude {
+                top = annotationOption
+            }
+            if left == nil || left.point.coordinates.longitude > annotationOption.point.coordinates.longitude {
+                left = annotationOption
+            }
+            if bottom == nil || bottom.point.coordinates.latitude > annotationOption.point.coordinates.latitude {
+                bottom = annotationOption
+            }
+            if right == nil || right.point.coordinates.longitude < annotationOption.point.coordinates.longitude {
+                right = annotationOption
+            }
+        }
+
+        let camera = mapboxMap.camera(
+            for: GeometryCollection(geometries: [top, left, bottom, right].compactMap(\.geometry)).geometry,
```

Comment:
I wonder if it would be a good idea to also expose a way for the user to get a camera for given list of annotations? Or maybe even geometry encompassing the annotations itself 🤔 

This would give developers a way to run custom animations for showing the view annotations, to plug this animation into a complex animation etc.

Reply:
We have to align this new API with Android SDK. May be they already have something similar.